### DISCUSSION TITLE
Challenge closes tonight at midnight

### DISCUSSION THREAD

**Discussion by aboullaite** [geeksblabla/blanat#291] [Announcements]
The challenge will close tonight, 29th January 23:59:59pm. PR made before will still be merged and reviewed. Prs merged after that will automatically be closed.

Thank you all for your participation. We will announce the results next week

**Comment by anassajaanan** [comment]
I want to extend my sincerest thanks to the geeksblabla community for organizing this challenge. It was an amazing experience that not only pushed the boundaries of what we can achieve with technology but also fostered a spirit of collaboration and learning among all the participants. Throughout this challenge, we all won in our own ways by gaining new insights, sharpening our skills, and coming together as a community to tackle a fascinating problem. A big thank you to everyone involved - the organizers for setting up such a compelling challenge, and to all my fellow participants for making it a rich, competitive, and educational experience. I'm looking forward to future challenges and opportunities to learn and grow together. Thank you, geeksblabla community!

**Reply by aboullaite** [reply]
Thanks a lot @anassajaanan. That is the spirit indeed 🙌 we all won and we all learned something new. For is the team, we learned a lot on how we can improve next time. Thanks for your feedback and we would be happy to have you in a geeksblabla episode about this challenge if you like
